src/less/pricingEstimate.less

```diff
@@ -80,7 +81,8 @@
 
   #priceEstimateContainer {
     box-shadow: 7px 0 7px -7px rgba(0, 0, 0, 0.2);
-
+    position: sticky;
```

Comment:
did you mean to add this for every price estimator? that's fine if you did but is there any possible problem that could come out of this? AKA will this break anyone?